andela/travel_tool_front
# #160751476 Restyle toasts to match style guide

#### What does this PR do?
* it changes the toast styles

#### Description of Task to be completed?
* Change the toaster

#### How should this be manually tested?
* Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
* Change into the project directory
* Check out this branch with `git checkout ch-show-notification-toast-160751466`
* Start the server `make start`
* Setup the backend by following this [README.md](https://github.com/andela/travel_tool_front)
* In a browser, navigate to `localhost:3000`
* Perform an action that requires you to get a notification(create a request, approve a request)
* You should see the redesigned toaster

#### Any background context you want to provide?
* [Link To Toast Mock-up](https://projects.invisionapp.com/d/main/default/#/console/15027511/312473868/preview)

#### What are the relevant pivotal tracker stories?
* [#160751476](https://www.pivotaltracker.com/story/show/160751476)

#### Screenshots (if appropriate)
<img width="1424" alt="screen shot 2018-09-27 at 3 00 43 pm" src="https://user-images.githubusercontent.com/27569442/46151306-5e56ce00-c266-11e8-958d-b808c11e9b61.png">



#### Questions:
* N/A
